classes/invitation/invitations/BaseInvitation.php

```diff
@@ -252,4 +255,8 @@ public function getId(): int
     {
         return $this->id;
     }
+    public function getData($key)
```

Comment:
I am not sure why that is needed - Is the `$key` parameter here necessary? Also is it a good idea to expose the `$key`  parameter like so?

Reply:
i have to create getData function to get data to columns mirror to [$element->getData($columnId);](https://github.com/pkp/pkp-lib/blob/main/classes/controllers/grid/DataObjectGridCellProvider.php#L73)